.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
```

Comment:
You could also add github actions like in operator-rs: https://github.com/stackabletech/operator-rs/blob/main/.github/dependabot.yml